.gitignore

```diff
@@ -1,3 +1,3 @@
 /target/
 .*
-
+*.iml
```

Comment:
I don't mind adding it here.. since I started using global .gitignore I don't have to care about these "issues".